src/inmanta/const.py

```diff
@@ -168,11 +168,40 @@ class LogLevel(Enum):
     TRACE = 3
     NOTSET = 0
 
+    @property
+    def to_str(self) -> "StrLogLevel":
+        """
+        Convert an integer formated log level into a string formated log level
+        """
+        return StrLogLevel[self.name]
+
+
+@stable_api
+class StrLogLevel(str, Enum):
```

Comment:
The addition of this class might seem strange, as it is not use for now.  This will actually be consumed by `inmanta-lsm`, to fix the openapi doc of some of its routes.